Makefile

```diff
@@ -89,10 +89,10 @@ $(C_BUILDDIR)/eeprom.o: CFLAGS = -O1 -mthumb-interwork -Wimplicit -Wparentheses
 C_SRCS := $(wildcard $(C_SUBDIR)/*.c $(C_SUBDIR)/*/*.c)
 C_OBJS := $(patsubst $(C_SUBDIR)/%.c,$(C_BUILDDIR)/%.o,$(C_SRCS))
 
-ASM_SRCS := $(wildcard $(ASM_SUBDIR)/*.s)
+ASM_SRCS := $(wildcard $(ASM_SUBDIR)/*.s $(ASM_SUBDIR)/*/*.s)
 ASM_OBJS := $(patsubst $(ASM_SUBDIR)/%.s,$(ASM_BUILDDIR)/%.o,$(ASM_SRCS))
 
-DATA_ASM_SRCS := $(wildcard $(DATA_ASM_SUBDIR)/*.s $(DATA_ASM_SUBDIR)/**/*.s)
+DATA_ASM_SRCS := $(wildcard $(DATA_ASM_SUBDIR)/*.s $(DATA_ASM_SUBDIR)/**/*.s $(DATA_ASM_SUBDIR)/**/**/*.s)
```

Comment:
Is this necessary? Or does the `**` wildcard mean "any number of directories deep" anyways? (if the latter it should probably also be changed for `ASM_SRCS`

Reply:
I think the wildcard only goes down a single level

Reply:
Actually, not sure what the difference between `*` and `**` is

Reply:
It seems `**` is just interpreted as `*` in normal make: https://stackoverflow.com/a/52534636
`**` as a glob for multiple directories seems to only be available in makepp or zsh: https://www.math.colostate.edu/~yzhou/computer/writemakefile.html#Using%20Wildcards (last bullet point before the next section)

Reply:
good to know, we should probably choose one over the other to normalize behavior on all build systems.